packages/stream-tests/src/__tests__/cross-node-indexing.test.ts

```diff
@@ -93,19 +97,27 @@ describeIfV3ShouldPass.each(envs)(
     beforeAll(async () => {
       await pgSetup()
 
-      ipfs1 = await createIPFS()
-      ipfs2 = await createIPFS()
+      ipfs1 = await createIPFS(undefined, undefined, {
```

Comment:
just an observation: seems like having multiple constructors would be nice some day (passing undefined like this just hurts lol)